Automattic/jetpack
# Admin Page: fix the display of the markdown for comments settings

Fixes #33878

## Proposed changes:

Since Markdown for comments isn't a module but a setting, it uses a different toggle, and it shows in the display of the settings. Let's adjust the look of the label to match the others.

**Before**

<img width="613" alt="Screenshot 2023-10-31 at 12 56 48" src="https://github.com/Automattic/jetpack/assets/426388/792add50-db45-4a8f-af84-44bace34bb0b">

**After**

<img width="701" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/fbf9b5b6-45d0-426f-86ad-a20b482679e2">

> **Note**
> This PR also corrects capitalization of the "Likes" feature in the comment likes toggle label.

### Other information:


## Jetpack product discussion

* p1697057905295799-slack-C0172LTKWF4

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* On a site connected to WordPress.com,
* Go to Jetpack > Settings > Discussion
* Ensure the Markdown settings' label matches the ones above and below.
